Competition2018/src/main/java/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -291,9 +290,8 @@ public void setupElevatorCommands(OperatorInterface oi, CalibrateElevatorTicksPe
     }
 
     @Inject
-    public void setupClimberCommands(OperatorInterface oi, AscendClimberCommand ascend, AscendLowPowerCommand ascendSlowly) {
+    public void setupClimberCommands(OperatorInterface oi, AscendClimberCommand ascend) {
         oi.driverGamepad.getifAvailable(1).whileHeld(ascend); // a
-        oi.driverGamepad.getifAvailable(2).whileHeld(ascendSlowly); // b
```

Comment:
I think we still want this one; it would be helpful for ensuring the hook is sturdily engaged before winching at full power.

Reply:
done